### ISSUE TITLE
New navigation

### ISSUE THREAD

**Issue by emersonthis** [flattenthecurve/guide#214]
We did a short term update to the homepage in #146 but the wireframe included modifications to the navigation which have not happened yet...

<img width="590" alt="Screen Shot 2020-03-22 at 4 57 38 PM" src="https://user-images.githubusercontent.com/1728139/77266995-6e364f00-6c5e-11ea-9b3e-8e660ddec835.png">

We _may_ want to stall on this while some additional content reorganization is still happening... Check with @mverzilli before starting work on this.

**Comment by jmcmurry** [comment]
This is a better homepage but act and prepare needs to be front and center especially now that people are getting the picture that this is a problem.

**Comment by emersonthis** [comment]
@jmcmurry this issue is only about the navigation at the top. Are you commenting on that specifically? (The result of that diagram is left over from another issue)

**Comment by emersonthis** [comment]
This is old news.
